index.html

```diff
@@ -1,89 +1,96 @@
-                <button  data-id="endereco">Endereço</button>
-                <button  data-id="finalizacao">Finalização</button>  <br><br></br><br>
-            </div>
-            <!--Primeira página-->
+        <div class="abas-navegacao">
+            <button class="botaoInicio" data-id="dados" data-open>Dados</button>
+            <button class="botaoEndereco" data-id="endereco" disabled>Endereço</button>
+            <button class="botaoFinal" data-id="finalizacao" disabled>Finalização</button> <br><br></br><br>
+        </div>
+        <!--Primeira página-->
         <div class="area-form">
-                <section id="dados">
-                    <form>
-                        <label for="name">Nome*</label>
-                        <input type="text" id="name" placeholder="Digite seu nome"><br>
-                        
-                        <label for="dataNas">Data de nascimento*</label>
-                        <input type="date" id="date"><br>
+            <section id="dados">
+                <form method="post" name="forms">
+                    <label>Nome*</label>
+                    <input type="text" id="name" placeholder="Digite seu nome" autocomplete = "off"><br>
 
-                        <labeld for="cpf">CPF*</labeld>
-                        <input type="text" id="cpf" placeholder="000.000.000-00"><br><br>
+                    <label>Data de nascimento*</label>
+                    <input type="date" id="date" placeholder="YYYY-MM-DD" min="0000-00-00" max="2022-05-19"><br>
 
-                        <div class="button">
+                    <labeld>CPF*</labeld>
+                    <input class="cpf" type="text" id="cpf" placeholder="000.000.000-00" required autocomplete="off"maxlength="14"><br><br>
 
-                            <button type="button" data-id="endereco" onclick="validaDadosPreenchidos()">Avançar</button>
+                    <div class="button">
 
-                        </div> 
+                        <button id="botaoAvancar" type="button" data-id="endereco" onclick="validaNome()" value="" disabled="true">Avançar</button>
 
+                    </div>
 
+                </form>
+
+            </section>
 
-                    </form>
-                </section>
-            
             <!--Segunda página-->
-            
-                <section id="endereco">
-                    <form method="get">
-                        <label for="cep">Digite seu CEP</label>
-                        <input type="text" name="cep id="cep" placeholder="Digite seu CEP"><br><br>
 
-                        <label for="logradouro">Logradouro*</label>
-                        <input type="text" id="logradouro"><br>
+            <section id="endereco">
+                <form method="get" name="forms2">
+                    <label>Digite seu CEP</label>
+                    <input type="text" name="cep" id="cep" placeholder="Digite seu CEP" maxlength="9"
```

Comment:
Da pra identar melhor tags HTML com muitos atributos.
```suggestion
                    <input
                        type="text"
                        name="cep"
                        id="cep"
                        placeholder="Digite seu CEP"
                        maxlength="9"
```